seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumComJSRunner.scala

```diff
@@ -51,7 +61,18 @@ class SeleniumComJSRunner(browserProvider: SeleniumBrowser,
 
         case msg: String =>
           browser.processConsoleLogs(console)
-          msg
+          if (msg.startsWith(MESSAGE_TAG)) {
+            val taglessMsg = msg.substring(MESSAGE_TAG.length)
+            taglessMsg.replace("\u0007\u0007", "\u0000").map {
+              case '\u0000' => '\u0007'
+              case '\u0007' => '\u0000'
+              case c        => c
+            }
```

Comment:
The current on is quite fragile. I replaced the decoding with regex replace that does it in one pass, it should be clearer. 

I tried to avoid the the loops because i could not find a scheme that would work well in Scala and JS at the same time.